src/Nethermind/Nethermind.Db.Rocks/DbOnTheRocks.cs

```diff
@@ -313,10 +313,16 @@ public Span<byte> GetSpan(byte[] key)
 
         UpdateReadMetrics();
 
-        return _db.GetSpan(key);
+        Span<byte> span = _db.GetSpan(key);
+        GC.AddMemoryPressure(span.Length);
```

Comment:
Will cause an exception for zero-length span e.g. initial run (no DB yet created)

Reply:
Added check to avoid this